### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by kaus02**
Added some more organisations worth mentioning.

### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/open-source-organizations/index.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ items:
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
  - OSGeo/osgeo
+ - Airbnb/github.com/airbnb
+ - PayPal/github.com/paypal
+ - WalmartLabs/github.com/walmartlabs
```
Comment (MikeMcQuaid) [inline collections/open-source-organizations/index.md]: These all need to be without the `github.com/` part.

**Comment by masterpass78** [comment]
Need my money

**Comment by Saagat** [comment]
wiche money man

File: collections/open-source-organizations/index.md (hunk 1 above)
Comment (kaus02) [reply collections/open-source-organizations/index.md]: There are several organisations that don't have their github pages like PayPal, Airbnb, WalmartLabs, Evernote, Pinterest.

Comment (MikeMcQuaid) [reply collections/open-source-organizations/index.md]: @kaus02 This syntax as-is won't work. You need to omit the `github.com/`. Would you like me to reopen this so you can push a commit to address this feedback?

**Pull request closed without merging**